tjpcov/wigner_transform.py

```diff
@@ -432,22 +400,18 @@ def wigner_d_parallel(s1, s2, theta, ell, ncpu=None, l_use_bessel=1.0e4):
 #  to use the faster version, but if weird results appear, it might be this
 #  function has some remaining bug.
 def bin_cov(r, cov, r_bins):  # works for cov and skewness
-    """
-    Function to apply the binning operator. This function works on both one
-    dimensional vectors and two dimensional covariance covrices.
-
-    Parameters
-    ----------
-    r:
-        theta or ell values at which the un-binned vector is computed.
-    cov:
-        Unbinned covariance. It also works for a vector of C_ell or xi
-    r_bins:
-        theta or ell bins to which the values should be binned.
+    """Function to apply the binning operator.
 
-    Returns
-    -------
-    (array): Binned covariance or vector of C_ell or xi
+    This function works on both one dimensional vectors and two dimensional
+    covariance covrices.
+
+    Args:
+        r: theta or ell values at which the un-binned vector is computed.
+        cov: Unbinned covariance. It also works for a vector of C_ell or xi
+        r_bins: theta or ell bins to which the values should be binned.
+
+    Returns:
+        array: Binned covariance or vector of C_ell or xi
```

Comment:
Also very minor but I think array should be array_like

Reply:
Done.